Documentation/ceph-cluster-crd.md

```diff
@@ -18,7 +18,6 @@ Settings can be specified at the global level to apply to the cluster as a whole
 ### Cluster Settings
 - `dataDirHostPath`: The path on the host ([hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)) where config and data should be stored for each of the services. If the directory does not exist, it will be created. Because this directory persists on the host, it will remain after pods are deleted.
   - On **Minikube** environments, use `/data/rook`. Minikube boots into a tmpfs but it provides some [directories](https://github.com/kubernetes/minikube/blob/master/docs/persistent_volumes.md) where files can be persisted across reboots. Using one of these directories will ensure that Rook's data and configuration files are persisted and that enough storage space is available.
-  - If a path is not specified, an [empty dir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) will be used and the config will be lost when the pod or host is restarted. This option is **not recommended**.
```

Comment:
what's the difference in behavior here? what happens if you don't set a `dataDirHostPath` value?

Reply:
an empty `dataDirHostPath` is not supported anymore. It is required to get the provisioning pod to work with the new osd pods. However, it would be nice to put a better error message in place. The behavior is that the operator tries to start the osds, but fails. I'll open an issue for that.